experiments/pam/pam_cortx.c

```diff
@@ -49,14 +49,14 @@ static int login(pam_handle_t *pamh, const char *pUsername, const char *pPasswor
 {
     //Login Function for CSM Initialization.
 	pam_syslog(pamh, LOG_INFO, "Verify CSM User %s \n", pUsername);
-	const char *pUrl = "http://localhost:8101/api/v1/login?debug";
+	const char *pUrl = "http://localhost:28101/api/v1/login";
 	CURL *pCurl = curl_easy_init();
 	struct curl_slist *headers = NULL;
 	int res = -1;
 	static const char *header_file_name = "/var/log/response_headers.out";
```

Comment:
- `/var/log` does not seem a good location to these files; why can't we use `/tmp` instead?
- A randomized filename would help mitigate (not to fix, I believe) the issue pointed out by Codacy.  See `mkstemp(3)`.